### PR TITLE
consistence fixes, e.g. old methods removed, etc

### PR DESCRIPTION
**Pull request by struberg**
Parts have been taken over from mp-config.

### CODE REVIEW

File: spec/src/main/asciidoc/configaccessor.asciidoc [hunk 1]
```diff
@@ -44,7 +44,8 @@ This is basically a builder which has methods to refine the resolution, includin
 * `asList()` -- Declare the ConfigAccessor to return a List of the given Type.
 * `evaluateVariables(boolean evaluateVariables)`
 * `useConverter(Converter<T> converter)` -- Defines a specific {@link Converter} to be used instead of applying the default Converter resolving logic.
-* `withLookupChain(String... postfixNames)` -- sets a parameter for the resolution of a 'postfix' for the resolution
+* `withLookupSuffix(String postfixName)` -- sets a parameter for the resolution of a 'postfix' for the resolution
+* `withLookupSuffix(ConfigAccessor)` -- you can also use a `ConfigAccessor` to determine the 'postfix' for the resolution
```
Comment (Emily-Jiang) [inline spec/src/main/asciidoc/configaccessor.asciidoc]: withLookupSuffix(ConfigAccessor accessor)

File: spec/src/main/asciidoc/configaccessor.asciidoc [hunk 2]
```diff
@@ -44,7 +44,8 @@ This is basically a builder which has methods to refine the resolution, includin
 * `asList()` -- Declare the ConfigAccessor to return a List of the given Type.
 * `evaluateVariables(boolean evaluateVariables)`
 * `useConverter(Converter<T> converter)` -- Defines a specific {@link Converter} to be used instead of applying the default Converter resolving logic.
-* `withLookupChain(String... postfixNames)` -- sets a parameter for the resolution of a 'postfix' for the resolution
+* `withLookupSuffix(String postfixName)` -- sets a parameter for the resolution of a 'postfix' for the resolution
```
Comment (Emily-Jiang) [inline spec/src/main/asciidoc/configaccessor.asciidoc]: addLookupSuffix?

**Comment by struberg** [comment]
This PR is outdated, but the changes in it are still important.
We basically need to bring our documentation up2date.
Gonna ship a new PR.

**Comment by struberg** [comment]
I've applied the changes and assembled a new PR: i70

**Pull request closed without merging**